api/query/atoms.go

```diff
@@ -78,8 +78,10 @@ func (e AbstractExists) BindToRuns(runs ...shared.TestRun) ConcreteQuery {
 	queries := make([]ConcreteQuery, len(e.Args))
 	for i, arg := range e.Args {
 		var query ConcreteQuery
+		// For sequential + count, we pass all runs.
 		if _, isSeq := arg.(AbstractSequential); isSeq {
-			// For sequential, we pass all runs.
+			query = arg.BindToRuns(runs...)
+		} else if _, isCount := arg.(AbstractCount); isCount {
```

Comment:
Looks like count operator is only implemented at the root level. If I understand correctly, a case like
 exists: [{
          and: [
            {pattern: 'a'},
           {
          count: 5,
          where: {
            or: [{status: 'PASS'}, {status: 'OK'}],
          },
        }
      ]}]